Cesium3DTilesSelection/include/Cesium3DTilesSelection/TilesetViewGroup.h

```diff
@@ -0,0 +1,230 @@
+#pragma once
+
+#include <Cesium3DTilesSelection/TileLoadRequester.h>
+#include <Cesium3DTilesSelection/TileLoadTask.h>
+#include <Cesium3DTilesSelection/TileSelectionState.h>
+#include <Cesium3DTilesSelection/ViewUpdateResult.h>
+#include <CesiumUtility/CreditReferencer.h>
+#include <CesiumUtility/IntrusivePointer.h>
+#include <CesiumUtility/TreeTraversalState.h>
+
+#include <cstddef>
+#include <unordered_map>
+#include <vector>
+
+namespace Cesium3DTilesSelection {
+
+class Tile;
+class Tileset;
+class TilesetContentManager;
+class TilesetFrameState;
+
+/**
+ * @brief A group of views that select tiles from a particular {@link Tileset}
+ * together.
+ *
+ * Create an instance of this class and pass it repeatedly to
+ * {@link Tileset::updateViewGroup} to select tiles suitable for rendering the
+ * tileset from a given view or set of views.
+ */
+class CESIUM3DTILESSELECTION_API TilesetViewGroup final
```

Comment:
I probably should have looked at this file earlier because it's the whole point of the PR... but I was surprised that this "view group" doesn't actually have any sort of "view" objects. So I'm not sure I understand the relationship between views in a runtime client -- i.e. literal cameras -- and a view group here. How would client views get associated with a specific view group, or multiple?

Reply:
Maybe this is my <7hrs of sleep talking, but if I replace the name with just `TilesetView` it clicks more easily in my head...

Reply:
Ok, after re-reading the README description for this class, I think my source of confusion was that I expected the `ViewState`s to be associated with one group, which is not necessarily true. The `ViewState`s (which are what "views" conceptually are, from my understanding), are decoupled from the `TilesetViewGroup`, because a client could dynamically change the `ViewState`s that are viewing a tileset...

The `TilesetViewGroup` is not a literal group of views, but rather manages the state of tiles in the context of some group of views... but it also doesn't have any actual internal knowledge of what views influenced it, it simply exists and the client has to keep track of it... is that a fair assessment? 🤣

Reply:
Haha yes that's right. Just like `Tileset` previously didn't have a list of views.

There's no reason to force the user to manage a list of ViewStates owned by the view group and/or tileset when they're almost certainly going to create them from their own view definitions every frame anyway.

Reply:
Gotcha, I think I get it! I might propose tweaking the doc for a bit more clarity, but it's mostly my brain being a little too literal and silly 😋